source/jwtd/jwt_openssl.d

```diff
@@ -75,9 +75,9 @@ version(UseOpenSSL) {
         import std.exception : assertThrown;
 
         assertThrown(getESKeypair(0, "key"));
-        assertThrown(getESKeypair(NID_secp256k1, "bogus_key"));
-        assertThrown(getESKeypair(NID_secp256k1, null));
-        assertThrown(getESKeypair(NID_secp256k1, private256));
+        assertThrown(getESKeypair(NID_X9_62_prime256v1, "bogus_key"));
```

Comment:
I would highly recommend extracting this to a constant. Adding the comment about the equivalence of prime256v1, secp256r1, and P-256 is probably a good idea too since the definition for ES256 in the JWT RFC 7518 refers to P-256.

Reply:
Is not NID_X9_62_prime256v1 such a constant?

There's a lot of confusion about the name of the algorithm -- secp256r1, prime256v1 are synonymous.  secp256k1 is a different animal altogether.

Anybody messing about in this code ought to have this understanding.